00-Raster-Structure.Rmd

```diff
@@ -353,44 +346,107 @@ GDALinfo("NEON_RemoteSensing/HARV/DSM/HARV_dsmCrop.tif")
 
 ```
 
-## Bad data values
-
-Bad data values are different from NoData values. Sometimes a raster's metadata
-will tell us a value range of values for the raster. Sometimes, we need to use
-some practial scientific insight as we examine the data - just as we would for field
-data. 
+2. **Bad data values:** Bad data values are different from NoData values. 
```

Comment:
So my thought with that, and this section, is that if you are just looking for high/ low values you'd see that in the section that we previously did looking at min/max.  We'd catch them at that point and have to talk about them if our max value is 40,000m.  Bad values is a concept that needs to be introduced but might not be easy to introduce with this dataset.

Having the discussion, even though they don't see that bad data values might be worthwhile.  Could bring in a challenge of having bad data values in a later dataset.